docs/technical_reference/unit_models/reverse_osmosis_1D.rst

```diff
@@ -4,6 +4,7 @@ This reverse osmosis (RO) unit model
    * is 1-dimensional
    * supports a single liquid phase only
    * supports steady-state only
+   * supports flat-sheet and spiral-wound module designs
```

Comment:
I am guessing the 1d documentation references the 0d documentation for covering equations, hence the exclusion of the alternative equations for effective membrane area. Correct?

Reply:
Yes, since the membrane area equations weren't originally in the 1D docs I kept it that way